sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -1543,8 +1543,10 @@ public virtual Response SetAccessPolicy(IEnumerable<TableSignedIdentifier> table
         /// <returns><see cref="Response{T}"/> containing a <see cref="IReadOnlyList{T}"/> of <see cref="Response"/>.
         /// Each sub-response in the collection corresponds to the <see cref="TableTransactionAction"/> provided to the <paramref name="transactionActions"/> parameter at the same index position.
         /// Each response can be inspected for details for its corresponding table operation, such as the <see cref="Response.Headers"/> property containing the <see cref="ResponseHeaders.ETag"/></returns>
-        /// <exception cref="RequestFailedException"/> if the batch transaction fails./>
-        /// <exception cref="InvalidOperationException"/> if the batch has been previously submitted.
+        /// <exception cref="RequestFailedException"> Thrown when the batch transaction operation fails.</exception>
+        /// <exception cref="ArgumentNullException"> Thrown when <paramref name="transactionActions"/> is <c>null</c>.</exception>
+        /// <exception cref="InvalidOperationException">Thrown if <paramref name="transactionActions"/> contains no items.</exception>
+        /// <exception cref="InvalidOperationException"> Thrown if the batch has been previously submitted.</exception>
```

Comment:
Could there be any issue or confusion if `InvalidOperationException` is repeated?

Reply:
It'll appear in docs twice, but I think that's more clear than having it listed once with a big sentence describing all of the different scenarios.   It is also consistent with how we generally do docs for parameter validation, where things like `ArgumentNullException` get triggered by several parameters.